woocommerce/woocommerce-admin
# Tax Task: Remove check for general settings requesting

https://github.com/woocommerce/woocommerce-admin/pull/4647, required general settings to be preloaded. Unfortunately, the tax task relied on general settings being fetched and populated before conditionally checking some logic. Because that request has been eliminated, the conditional logic was never reached in the code.

This PR removes that check. It also fixes a regression on the Success screen's "Yes please" button.

#### Before

![Screen Shot 2020-06-23 at 12 30 53 PM](https://user-images.githubusercontent.com/1922453/85348609-389c4d00-b550-11ea-9806-3076b2961e6c.png)

#### After

![Screen Shot 2020-06-23 at 12 33 04 PM](https://user-images.githubusercontent.com/1922453/85348613-3a661080-b550-11ea-8da7-9f1d953878b8.png)

### Detailed test instructions:

1. On a jetpack connected site, go to the Task List.
2. Click on the tax task.
3. See the Success screen. Previously nothing would have happened.



### Changelog Note:

none: fixes unreleased error.
